docs/reference/components/sources/docker_logs.cue

```diff
@@ -86,6 +86,12 @@ components: sources: docker_logs: {
 				[Docker `journald` driver](\(urls.docker_logging_driver_journald))
 				drivers.
 				""",
+			"""
+				To avoid collecting logs from itself, the Docker source does not
+				collect logs from Vector containers. Please consider using the
+				[`vector` source](\(urls.vector_source)) if you want to collect logs
+				from other Vector containers.
+				""",
```

Comment:
Does this change in semantics make this a breaking change? If so, make sure it's marked in the title.

Reply:
I don't think this behaviour was documented previously. Anyway, I've marked it.

Reply:
Do you mean it will not collect logs from _any_ vector container? Or just itself?

Reply:
From any vector container. Currently, there is two filters in `exclude_vector`:
- `match_hostname`: hostname inside a container is usually a short ID of that container, so we use this info to find out which container current vector process is in. 
- `match_image`: exclude all vector images.

If `match_hostname` works well enough, we could remove `match_image`.
cc @ktff

Reply:
:+1: I think that would be nice if we could remove `match_image`. I can see a case where a user is running vector as an agent on a server, in docker, to collect all docker logs and then also another `vector` instance as an aggregator, also in docker, on the same server where it'd be useful to have the agent forward the aggregator's logs.

However, it also seems like that behavior hasn't changed so I'm happy to punt on this to a separate issue if we want.

Reply:
Also, I think we can avoid marking this change as a breaking change as it isn't changing, we just hadn't documented it.

Reply:
I'm in favour of removing `match_image` too, I've removed it.